### DISCUSSION TITLE
Django integration

### DISCUSSION THREAD

**Discussion by biofsphere** [squidfunk/mkdocs-material#3154] [Ideas]
I have been considering to integrate mkdocs-material in a Django project. Have anyone else considered the same and if yes, how did you address the problem? Build first then drag the static generated doc to a django template and tag it accordingly? Seem laborious, though.  I know I could host docs completely apart from my Django project, but just for sake of having it all in a nicely built Django project, any ideas?

**Comment by squidfunk** [comment]
Django and MkDocs solve entirely different use cases. You might be better of building two separate projects and hosting them on different subdomains or paths with a reverse proxy.
